### DISCUSSION TITLE
RSS Feed for a Mastodon hashtag?

### DISCUSSION THREAD

**Discussion by metaperl** [mastodon/mastodon#25497] [Q&A] [answered]
Hello, I would like an RSS for [post with a certain hashtag](https://mastodon.social/tags/python). It this possible?

**Comment by andypiper** [comment]
This is not implemented in Mastodon core, so you would need to check to see whether there's an existing issue or pull request for this idea.

**Comment by stefanhayden** [comment, marked as answer]
I think this now works?

https://mastodon.social/tags/python.rss

triggers an rss file download in my browser

**Reply by dmathieu** [reply]
It actually looks like this already worked when this discussion was opened.
https://github.com/mastodon/mastodon/pull/7259
